vagrant/bootstrap.sh

```diff
@@ -11,6 +11,9 @@ server {
 	listen 80 default_server;
 	listen [::]:80 default_server;
 
+	# set client body size
+    client_max_body_size 5M;
```

Comment:
We should consider that the `bootstrap.sh` script runs only once when the vm starts the first time. This means that changes will only apply after `vagrant destroy && vagrant up`. But this behaviour is definitely sufficient.